domains/emissions/Emissions.API/API/Services/EmissionDataService.cs

```diff
@@ -24,9 +25,28 @@ public async Task<EmissionsDataResponse> GetEmissionsPerHour(DateTime dateFrom,
         throw new Exception("EDS Emissions query failed");
     }
 
+    public async Task<DeclarationProduction> GetDeclarationProduction(DateTime dateFrom, DateTime dataTo)
+    {
+        var result = await httpClient.GetFromJsonAsync<DeclarationProduction>(GetQuery(dateFrom, dataTo));
+        if (result != null)
+        {
+            return result;
+        }
+
+        throw new Exception($"EDS declarationproduction query failed. query:{GetQuery(dateFrom, dataTo)}");
+    }
+
+    string GetQuery(DateTime dateFrom, DateTime dateTo)
+    {
+
+        return "datastore_search_sql?sql=SELECT \"HourUTC\", \"PriceArea\", \"Version\", \"ProductionType\", \"ShareTotal\" " +
```

Comment:
Why do we need sql literals?
Assuming we do, can they not be written in a more readable way without all the \ escapes? Maybe as .sql files included in the project?

Reply:
Tried using string literals to avoid \ but that made the query pretty much unreadable